src/AccountTile/AccountTile.js

```diff
@@ -0,0 +1,206 @@
+      onRequestClose={close}
+      targetContainerStyles={{
+        width: width,
+        display: 'block'
+      }}
+      placement="bottom-end"
+      targetEl={
+        // ---------- Account Tile ----------
+        <StyledAccountTile
+          onClick={toggleIsOpen}
+          onKeyPress={toggleIsOpen}
+          open={isOpen}
+          tabIndex="0"
+        >
+          <StyledContentWrapper>
+            <StyledAvatarContainer>
+              <Avatar src={userThumbnail.url} style={{ width: 60, height: 60 }}>
+                {!userThumbnail.url && userThumbnail.letters}
+              </Avatar>
+              {orgName && (
+                <Avatar
+                  src={orgThumbnail.url}
+                  style={{ width: 30, height: 30 }}
+                >
+                  {!orgThumbnail.url && orgThumbnail.letters}
+                </Avatar>
+              )}
+            </StyledAvatarContainer>
+            <StyledTextWrapper>
+              {orgName && (
+                <StyledP title={orgName} demi>
+                  {orgName}
+                </StyledP>
+              )}
+              <StyledP title={user.fullName || fullName}>
+                {user.fullName || fullName}
+              </StyledP>
+              <StyledP title={user.username} small>
+                {user.username}
+              </StyledP>
+            </StyledTextWrapper>
+          </StyledContentWrapper>
+          <StyledIconWrapper>
+            {authStatusIndicator}
+            {actions.length !== 0 && <HandleVerticalIcon scale={16} />}
+          </StyledIconWrapper>
+        </StyledAccountTile>
+      }
+    >
+      {/* Account item dropdown menu */}
+      <Menu>{actions.length !== 0 && menuItems}</Menu>
+    </Popover>
+  );
+};
+
+AccountTile.propTypes = {
+  /** Actions to be included in the dropdown. Each action should be structured: { label: 'Action Label', onClick: someMethod } */
+  actions: PropTypes.array,
+  /** The ArcGIS user object.*/
+  user: PropTypes.object.isRequired,
+  /** Is the user currently logged in or has their authentication expired? */
+  isAuthenticated: PropTypes.bool,
+  /** Name of the user's organization */
+  orgName: PropTypes.string,
+  /** Style prop to define the width of the tile.*/
+  width: PropTypes.string,
+  /** Text to show when the account is authenticated.*/
+  authenticatedText: PropTypes.string,
+  /** Text to show when the account has expired (is not authenticated).*/
+  expiredText: PropTypes.string,
+  /** Object containing thumbnail url and fallback text: { url: 'thumbnail.com', letters: 'JH' }.*/
+  userThumbnail: PropTypes.object.isRequired,
+  /** Object containing thumbnail url and fallback text: { url: 'thumbnail.com', letters: 'E' }.*/
+  orgThumbnail: PropTypes.object.isRequired
+};
+
+AccountTile.defaultProps = {
+  actions: [],
+  isAuthenticated: false,
+  orgName: '',
+  width: '100%',
+  authenticatedText: 'Account authenticated',
+  expiredText: 'Authentication expired'
```

Comment:
Would like to hear other opinions on this, but would it be more user-friendly to default these instead to `Signed in` and `Session expired`?

Reply:
Those defaults would be more logical, I'll make that change.